.github/workflows/release.yml

```diff
@@ -21,13 +36,29 @@ jobs:
       version: ${{ steps.parse.outputs.version }}
       runtime: ${{ steps.parse.outputs.runtime }}
     steps:
-      - uses: actions/checkout@v3
+      - name: checkout HEAD
+        uses: actions/checkout@v4
+        if: ${{ env.DRY_RUN }}
         with:
-          ref: ${{ github.ref }}
+          fetch-depth: 0
+      - name: check out tag
+        uses: actions/checkout@v4
+        if: ${{ !env.DRY_RUN }}
+        with:
+          ref: ${{ steps.ref.outputs.ref }}
       - id: parse
         name: Parse ref
         shell: bash
-        run: ./scripts/parse_ref.sh ${{ github.ref }} >> ${GITHUB_OUTPUT}
+        run: |
+          if [[ "${DRY_RUN}" == 'true' ]]
+          then
+            CRATE="${{ inputs.crate || 'containerd-shim-wasmtime' }}"
```

Comment:
Why do we just dry-run the wasmtime shim?

Reply:
Just simplicity. Could make it random. WDYT?

Reply:
I think we should dry run the core crate, and one of the runtime shim implementation.